oeps/processes/oep-0054-core-contributors.rst

```diff
@@ -0,0 +1,146 @@
+
+Questions about the Program can be directed to the tCRIL administrator(s) via the ``#core-contributors`` Slack room
+in the `Open edX Slack <http://openedx-slack-invite.herokuapp.com/>`_.
+
+.. _selecting new members:
+
+Adding New Core Contributors
+----------------------------
+
+The Open edX Core Contributor Program grants specific types of access rights to Open edX community members
+which enables them to further the development and adoption of the Open edX platform. CCs earn
+these rights by sustained, active participation in the Open edX community and by following the “3 Cs”:
+Commitment, Conduct, Caliber. Core Contributors follow the `Code of Conduct
+<https://open.edx.org/code-of-conduct/>`_, are respectful of other community members, are polite and
+welcoming to all in the community, and have shown the ability to deliver high quality work over time.
+
+Current project CCs are always on the lookout for project contributors who demonstrate the
+3 Cs and have been an overall active community member for some time. CCs, in conjunction work with the
+Program administrator(s), identify and nominate new CCs for CC roles that are ready to be expanded.
+When a new Core Contributor role is being defined and piloted, however,
+nominations should proceed only once the Program Administor is confident that new Core
+Contributors in that role can be supported.
+
+New Core Contributor Nomination Process
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The process for nomination is as follows:
+
+#. An existing CC ("sponsor") choses to nominate a community member ("candidate") for a new CC position
+#. The sponsor posts a new public thread on Discourse, in the `Core Contributor
+   discussion space <https://discuss.openedx.org/c/working-groups/core-contributors/36>`_, that
+   introduces the candidate, describes in a few paragraphs why they are suited to
+   join the program, and provides links to examples of previous Open edX work (for example, pull requests).
+   The post must mention the date when the comment period ends, which is one month from the start of the thread.
+#. The sponsor posts a link to the Discourse post in the ``#core-contributors`` Slack room,
+   as well as any other Slack rooms relevant to the CC role being discussed. The message must indicate
+   the dates of the comment period.
+#. During the comment period, other CCs should weigh in on the thread. If a CC has concerns they feel
+   cannot be raised publicly, they should consult with the Program Administrator to determine how to
+   proceed. Remember to leave constructive criticism - if a CC candidate is not ready yet, provide some
+   tips as to how they could make it into the program.
+#. Over the course of the comment period, the sponsor should respond to all comments that come in. It
+   may also be necessary to periodically nudge other CCs to respond to the thread.
+#. At the end of the comment period, if the candidate is approved, the candidate should work with the Program
```

Comment:
I think there needs to be some number of declarative "yes" votes, and no outstanding "no" votes. Does that feel right? How many "yes" votes make sense? How might it change with, say, 100 core contributors versus 5 product managers? I'm honestly inclined to say that after one month, 3-5 yes votes may be enough? Or should it be some % of the # of CCs in the role?

Reply:
Having no outstanding "no" and at least X "yes" sounds good. It can probably make sense to have at least 5 reviews/yes as a quorum, to make sure the application did get enough attention? Not getting too many reviews also mean something, like the way some PRs don't attract reviewers? It could be reviewers from any type of roles though, to help?

Reply:
5 is a good number. I do make it clear that throughout the comment period, it is the job of the candidate and the sponsor to drum up reviews.